samples/web-nodejs-sample/devfile.yaml

```diff
@@ -9,9 +9,6 @@ components:
   - alias: theia-editor
     type: cheEditor
     id: org.eclipse.che.editor.theia:1.0.0
```

Comment:
Should not `latest` be used instead of `1.0.0`? Last time when I tried to use `1.0.0` - commands did not work because of some bug that is already fixed in `next` and `latest`

Reply:
Yes, it most probably should be. Notice that this PR was opened on Apr 25 ;)